### DISCUSSION TITLE
Centering a Panel

### DISCUSSION THREAD

**Discussion by cvpfus** [spectreconsole/spectre.console#1333] [Q&A] [answered]
Is there any way to center a Panel?
![panel](https://github.com/spectreconsole/spectre.console/assets/47532266/33f88634-9df9-4628-b6e5-4704ebacaa65)

**Comment by patriksvensson** [comment, marked as answer]
`Panel` doesn't seem to implement `IAlignable`, but you should be able to wrap the panel in an `Align` widget to achieve the same thing.
